File: experiment_jp_test/j2pyraw/for__1__LONGEST_COMMON_SUBSEQUENCE_WITH_PERMUTATIONS_ALLOWED.java
class X { static void longestString ( String str1 , String str2 ) { int count1 [ ] = new int [ 26 ] , count2 [ ] = new int [ 26 ] ; String result = " " ; for ( int i = 0 ; i < 26 ; i ++ ) { for ( int j = 1 ; j <= Math . min ( count1 [ i ] , count2 [ i ] ) ; j ++ ) { result += ( char ) ( ' a ' + i ) ; } } System . out . println ( result ) ; }
 }

File: experiment_jp_test/j2pyrawpy/for__1__LONGEST_COMMON_SUBSEQUENCE_WITH_PERMUTATIONS_ALLOWED.py
#!/usr/bin/env python
""" generated source for module for__1__LONGEST_COMMON_SUBSEQUENCE_WITH_PERMUTATIONS_ALLOWED """
class X(object):
    """ generated source for class X """
    @classmethod
    def longestString(cls, str1, str2):
        """ generated source for method longestString """
        count1 = [None]*26
        count2 = [None]*26
        result = " "
        i = 0
        while i < 26:
            while j <= Math.min(count1[i], count2[i]):
                result += ((i))
                j += 1
            i += 1
        print result

